client-v2/src/components/FrontsEdit/ArticleFragmentForm.tsx

```diff
@@ -357,18 +359,45 @@ class FormComponent extends React.Component<Props, FormComponentState> {
           <RowContainer>
             <Row>
               <Col>
-                <ImageWrapper faded={imageHide}>
-                  <ConditionalField
-                    permittedFields={editableFields}
-                    name="primaryImage"
-                    component={InputImage}
-                    disabled={imageHide}
-                    criteria={imageCriteria}
-                    frontId={frontId}
+                {imageReplace && (
+                  <ImageWrapper faded={imageHide}>
+                    <ConditionalField
+                      permittedFields={editableFields}
+                      name="primaryImage"
+                      component={InputImage}
+                      disabled={imageHide}
+                      criteria={imageCriteria}
+                      frontId={frontId}
+                    />
+                  </ImageWrapper>
+                )}
+                {!imageReplace && (
+                  <ThumbnailEditForm
+                    style={{
```

Comment:
It would be nice to have the component be responsible for its own styles here, e.g. image={img} hide={imageHide} -- not a blocker, though!

Reply:
👍